src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Pledges.cshtml

```diff
@@ -19,6 +19,24 @@
 
 <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content">
     <div class="govuk-width-container">
+
+        @if (Model.PledgeClosedShowBanner)
+        {
+            <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
See previous comment - we need a generic solution to banners.

Reply:
Generic solution to the banner implemented.